### DISCUSSION TITLE
how can i use methods on my canvas

### DISCUSSION THREAD

**Discussion by smola14** [razztyfication/vue-drawing-canvas#27] [Q&A]
Hello, im using your canvas and everything works very well but i cant figure out how to use methods, for example reset canvas, can you advice me please? Thanks

**Comment by razztyfication** [comment]
Hello, maybe this [example](https://stackblitz.com/edit/vitejs-vite-j4dzsb?file=src/App.vue) will give you the idea how to use it with composition API Vue 3

for options API Vue 3 or with Vue 2 you can check the demo or simply use this.$refs on your method
